test/fixtures/dialects/oracle/alter_table_add_constraint.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE table_name
+ADD CONSTRAINT constraint_name
+FOREIGN KEY (column_name) REFERENCES other_table_name (other_column_name);
```

Comment:
I think this file could be merged with the other test file and be renamed to `alter_table.sql`

```suggestion
ALTER TABLE table_name
ADD CONSTRAINT constraint_name
FOREIGN KEY (column_name) REFERENCES other_table_name (other_column_name);

ALTER TABLE table_name RENAME COLUMN old_column_name TO new_column_name;
```

I'm a fan of grouping the test cases in fewer files where it makes sense. And yes I know we haven't always done that in this codebase but trying to get better at it! 😁